### PR TITLE
[13.0][MIG] account_skip_bank_reconciliation

### PR DESCRIPTION
**Pull request by LoisRForgeFlow**
Standard migration.

@ForgeFlow

### CODE REVIEW

File: account_skip_bank_reconciliation/models/account_reconcile_model.py [hunk 1]
```diff
@@ -1,16 +1,14 @@
...
     _inherit = "account.reconcile.model"
 
-    @api.multi
     def _apply_conditions(self, query, params):
-        query, params = super(AccountReconcileModel, self)._apply_conditions(
-            query, params
-        )
-        query += " AND account.exclude_bank_reconcile IS NOT TRUE"
+        query, params = super()._apply_conditions(query, params)
+        if "account_account account" in query:
```
Comment (MiquelRForgeFlow) [inline account_skip_bank_reconciliation/models/account_reconcile_model.py]: Maybe you could cherry-pick https://github.com/OCA/account-reconcile/commit/34fc6be367fac77d856b59c9f29571bdc80e9ad8 instead, which is more readable :octopus:

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by AaronHForgeFlow — APPROVED** [review]
Code review LGTM

File: account_skip_bank_reconciliation/models/account_reconcile_model.py (hunk 1 above)
Comment (LoisRForgeFlow) [reply account_skip_bank_reconciliation/models/account_reconcile_model.py]: adapted to your suggestion :)

**Pull request merged**